front/src/firebase/init.js

```diff
@@ -6,7 +6,7 @@ const firebaseConfig = require('./config.json');
 
 // Initialize Firebase
 firebase.initializeApp(firebaseConfig);
-firebase.auth().setPersistence(firebase.auth.Auth.Persistence.SESSION);
+firebase.auth().setPersistence(firebase.auth.Auth.Persistence.LOCAL);
```

Comment:
is there impact for "web" mode vs extension ?

Reply:
We need to have LOCAL for web like for extension